setup.py

```diff
@@ -11,10 +11,10 @@
     packages=find_packages(exclude=['docs', 'tests']),
     include_package_data=True,
     install_requires=[
-        'dimagi-superset==3.1.0.post1',
+        'dimagi-superset @ git+https://github.com/dimagi/superset.git@e0aa79525a',
         # Dependencies based on Superset 3.1.0 where applicable
         'Authlib==1.3.0',
-        'celery==5.2.7',
+        'celery==5.4.0',
```

Comment:
just checking that this change was intentional? we would be upgrading celery?

Reply:
This is just so that the Celery requirement of CommCare Analytics doesn't cause an unresolvable conflict with the [Celery requirement](https://github.com/dimagi/superset/blob/master/setup.py#L77) of Dimagi Superset (`"celery>=5.3.6, <6.0.0"`).

Reply:
is that the version currently installed on the environments (staging/production)?

Reply:
It's currently installed on Staging. I have not checked Prod.